ui/src/containers/Header/__snapshots__/Header.test.tsx.snap

```diff
@@ -65,6 +104,11 @@ exports[`Header should render the header component and find Search component 1`]
             <ModalContent variant=\\"small\\" isOpen={false} onClose={[Function: onClose]} className=\\"\\" showClose={true} id={[undefined]} actions={{...}} hasNoBodyWrapper={false} boxId=\\"pf-modal-part-0\\" labelId=\\"pf-modal-part-1\\" descriptorId=\\"pf-modal-part-2\\" title=\\"Search tips:\\" aria-label=\\"\\" aria-describedby=\\"\\" aria-labelledby=\\"\\" ouiaId={[undefined]} ouiaSafe={true} />
           </Portal>
         </Modal>
+        <Modal title=\\"Instructions to add Tasks or Pipelines:\\" isOpen={false} onClose={[Function: onClose]} width=\\"50%\\" className=\\"\\" aria-label=\\"\\" showClose={true} aria-describedby=\\"\\" aria-labelledby=\\"\\" id={[undefined]} actions={{...}} variant=\\"default\\" hasNoBodyWrapper={false} appendTo={[Function: appendTo]} ouiaSafe={true}>
+          <Portal containerInfo={{...}}>
+            <ModalContent isOpen={false} onClose={[Function: onClose]} width=\\"50%\\" className=\\"\\" showClose={true} id={[undefined]} actions={{...}} variant=\\"default\\" hasNoBodyWrapper={false} boxId=\\"pf-modal-part-1\\" labelId=\\"pf-modal-part-2\\" descriptorId=\\"pf-modal-part-3\\" title=\\"Instructions to add Tasks or Pipelines:\\" aria-label=\\"\\" aria-describedby=\\"\\" aria-labelledby=\\"\\" ouiaId={[undefined]} ouiaSafe={true} />
```

Comment:
Add a note that you need to contact hub owner to refresh the config of deployed Hub

Reply:
I have a though on the config refresh , when there are changes in config file and while reviewing  the PR  should be sureness from hub reviewer that PR would not the breaks existing production and config refresh would be done by the hub owner after PR merge instead of  forcing the contributor to contact the hub owner regarding config refresh , WDYT ?